### DISCUSSION TITLE
Is there a way to load a system font?

### DISCUSSION THREAD

**Discussion by royqh1979** [raysan5/raylib#2540] [Q&A] [answered]
Raylib's default font only have 95 glyphs (printable ascii  characters), so if you want to draw a text that contains non-ascii characters, you must load a font manually.

But it seems that raylib don't have a function that can load a system's font? (That is, load a font by it's name, not by it's fontfile's path).

Without this function, you have to distribute the app with a huge font file, or build a custom font file by yourself, even if the app just displays a few non-ascii chars. Or can we have any solution that's easier to use?

**Comment by ian1ilych** [comment]
My boy, there are no comments but find a solution.
I'm gonna figure it out.

**Comment by raysan5** [comment, marked as answer]
@royqh1979 I'm afraid this functionality is out-of-scope for raylib, it can be implemented in a higher-layer by users when required. raylib is a multi-platform library, there is no common solution for all the platforms. Requiring the font filepath is the right approach.

**Comment by vaguinerg** [comment]
#5007
